interfaces/builtin/basedeclaration.go

```diff
@@ -32,15 +32,103 @@ type: base-declaration
 authority-id: canonical
 series: 16
 revision: 0
-plugs:
-  network: true
 slots:
-  network:
-    allow-installation:
-      slot-snap-type:
-        - core
+  bluetooth-control:
+    deny-auto-connection: true
+  bluez:
+    deny-auto-connection: true
```

Comment:
yes, until we can set snap-declarations in the store we cannot really use deny-connection or deny-installation (or we can but then we need to turn off the respective checks)